lms/services/canvas_api.py

```diff
@@ -1,30 +1,45 @@
 import datetime
```

Comment:
I very much recommend:

 * Reading through the commits if you are interested how we got here
 * Then reading the file (not in diff format) to see what it's like now